go.mod

```diff
@@ -19,13 +19,17 @@ require (
 )
 
 require (
-	cloud.google.com/go/compute/metadata v0.2.3 // indirect
+	google.golang.org/genproto/googleapis/api v0.0.0-20230822172742-b8732ec3820d // indirect
+	google.golang.org/genproto/googleapis/rpc v0.0.0-20230822172742-b8732ec3820d // indirect
+)
+
+require (
```

Comment:
Why are there two `require` sections for the indirect dependencies in `go.mod`? Please run `go mod tidy` to fix this.

Reply:
`go mod tidy` didn't help here. I think the new `require` block was introduced just for the recent transitive dependencies, and it did not update the already added indirect dependencies. [This](https://go.dev/doc/go1.17#go-command) ([ref](https://stackoverflow.com/questions/69432877/why-there-are-two-require-blocks-in-go-mod-since-go-1-17)) seems to be the reason.
I'll update the go.mod manually.